### ISSUE TITLE
libinput - roadmap

### ISSUE THREAD

**Issue by proton-decay** [rusty-desktop/rusty-desktop-dev#3]
Discussion about road map for libinput.

**Comment by Drakulix** [comment]
@Meptl Any input from your side? I would otherwise just move my current bindings into this group and continue to maintain them.

**Comment by Meptl** [comment]
@Drakulix What's your take on using the bindgen generated files directly in the crate vs building them each time?

**Comment by Drakulix** [comment]
Currently I am building them each time (see https://github.com/Drakulix/input.rs/issues/2), but this is getting annoying and I am trying to figure out a way to check in up-to-date generated bindings into the repository, while still getting notified, when the generated file should be updated.

**Comment by Meptl** [comment]
Yeah, let's use your bindings; mine is mostly unmaintained anyway. On a similar note, if there were an option to rename/remove crates I wouldn't mind yours being the more formal "libinput-sys".
